### ISSUE TITLE
TypeError: expected np.ndarray (got float)

### ISSUE THREAD

**Issue by Apacchi88** [ExponentialML/ComfyUI_ELLA#10]
Using the provided workflow, but I am using Kijai's pruned model.

I am running latest Comfyui

I get the following error:

```
model_type EPS
Using pytorch attention in VAE
Using pytorch attention in VAE
clip missing: ['clip_l.transformer.text_projection.weight']
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
!!! Exception during processing !!!
Traceback (most recent call last):
  File "C:\SD\ComfyUI_windows_portable\ComfyUI\execution.py", line 151, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\SD\ComfyUI_windows_portable\ComfyUI\execution.py", line 81, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\SD\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_ezXY\autoCastPatch.py", line 299, in map_node_over_list
    return _map_node_over_list(obj, input_data_all, func, allow_interrupt)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\SD\ComfyUI_windows_portable\ComfyUI\execution.py", line 74, in map_node_over_list
    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\SD\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_ELLA\ella.py", line 100, in encode
    cond_ella = ella(cond, timesteps=torch.from_numpy(sigma))
                                     ^^^^^^^^^^^^^^^^^^^^^^^
TypeError: expected np.ndarray (got float)
```

**Comment by deniaud** [comment]
```
import numpy as np
sigma = np.array(sigma)
```

to 105 line in ella.py

**Comment by Apacchi88** [comment]
> ```
> import numpy as np
> sigma = np.array(sigma)
> ```
> 
> to 105 line in ella.py

Thanks, I tried that, Line 105 was: "# NOTE: names should be globally unique"
So I pasted it above that line. got the same error.

I then asked ChatGpt to help me as I don't know coding. 

It said I should put: `import numpy as np` in the beginning where the other modules are imported.
It then said to change line 102 (line 100 before adding import numpy as np) `cond_ella = ella(cond, timesteps=torch.from_numpy(sigma))` to -> `cond_ella = ella(cond, timesteps=torch.from_numpy(np.array(sigma)))`

Now it works ! 😀
